encoding/protobuf/protoc-gen-yarpc-go/internal/testing/testing.pb.yarpc.go

```diff
@@ -237,23 +242,489 @@ func (h *_SinkYARPCHandler) Fire(ctx context.Context, requestMessage proto.Messa
 	if requestMessage != nil {
 		request, ok = requestMessage.(*FireRequest)
 		if !ok {
-			return protobuf.CastError(emptySink_FireYARPCRequest, requestMessage)
+			return protobuf.CastError(emptySinkServiceFireYARPCRequest, requestMessage)
+		}
+	}
+	return h.server.Fire(ctx, request)
+}
+
+func newSinkServiceFireYARPCRequest() proto.Message {
+	return &FireRequest{}
+}
+
+func newSinkServiceFireYARPCResponse() proto.Message {
+	return &yarpcproto.Oneway{}
+}
+
+var (
+	emptySinkServiceFireYARPCRequest  = &FireRequest{}
+	emptySinkServiceFireYARPCResponse = &yarpcproto.Oneway{}
+)
+
+// AllYARPCClient is the YARPC client-side interface for the All service.
+type AllYARPCClient interface {
+	GetValue(context.Context, *GetValueRequest, ...yarpc.CallOption) (*GetValueResponse, error)
+	SetValue(context.Context, *SetValueRequest, ...yarpc.CallOption) (*SetValueResponse, error)
+	Fire(context.Context, *FireRequest, ...yarpc.CallOption) (yarpc.Ack, error)
+	HelloOne(context.Context, ...yarpc.CallOption) (AllServiceHelloOneYARPCClient, error)
```

Comment:
maybe rename the clients to be receiver/sender/receiveSender

Reply:
These should really stay as Client and Server to match grpc-go's naming.